### PR TITLE
LPS-127316 Fix event recorder popover layout

### PR DESCRIPTION
**Pull request by naraandrade**

### CODE REVIEW

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -433,6 +444,10 @@
 	}
 }
 
+.col-md-4 .calendar-portlet .popover.scheduler-event-recorder-popover {
```
Comment (interaminense) [inline modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:447]: Hey @naraandrade, Is it necessary to add `col-md-4` to this selector?

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss [hunk 2]
```diff
@@ -433,6 +444,10 @@
...
+	max-width: 14.5rem;
```
Comment (interaminense) [inline modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:448]: About this unit of measurement for `max-width`, why did you have to use the `rem`? It is relative to the parent's width and if the parent's width changes, that `max-width` will also be changed. Knowing this, wouldn't it be the case to set the `max-width` to `pixel`?

**Review by interaminense — COMMENTED** [review]
Hey @naraandrade, just a few questions about this PR

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (naraandrade) [reply modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:447]: Hi @interaminense :) Yes, this was the simplest way I identified to adjust the size of the popover when the calendar is used in a narrow space (a page with 2 columns 30/70, for example).

File: modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss (hunk 2 above)
Comment (naraandrade) [reply modules/apps/calendar/calendar-web/src/main/resources/META-INF/resources/css/main.scss:448]: Yes, I agree with you. I just repeated the default value that was defined in the popover class, but I will put it in pixels, thank you :)

**Review by rafaprax — APPROVED** [review]

**Comment by rafaprax** [comment]
ci:forward

**Review by interaminense — APPROVED** [review]
LGTM

**Comment by naraandrade** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
ci:forward

**Pull request closed without merging**